src/games/ninemensmorris/game.test.ts

```diff
@@ -28,76 +28,47 @@ it('should declare player 1 as the winner', () => {
   p1.connect();
 
   p0.moves.placePiece(0);
-  p1.moves.placePiece(9);
+  p1.moves.placePiece(8);
   p0.moves.placePiece(7);
-  p1.moves.placePiece(6);
-  p0.moves.placePiece(8);
-  p1.moves.placePiece(18);
-  p0.moves.placePiece(23);
   p1.moves.placePiece(15);
-  p0.moves.placePiece(14);
-  p1.moves.placePiece(22);
-  p0.moves.placePiece(13);
-  p1.moves.placePiece(21);
-  p0.moves.placePiece(20);
-  p1.moves.placePiece(2);
-  p0.moves.placePiece(3);
-  p1.moves.placePiece(4);
-  p0.moves.placePiece(1);
+  p0.moves.placePiece(12);
+  p1.moves.placePiece(14);
+  p1.moves.removePiece(12);
+  p0.moves.placePiece(6);
+  p0.moves.removePiece(14);
   p1.moves.placePiece(5);
-  p1.moves.removePiece(13);
-  p0.moves.movePiece(14, 13);
-  p1.moves.movePiece(9, 17);
-  p0.moves.movePiece(20, 19);
-  p1.moves.movePiece(21, 20);
-  p0.moves.movePiece(13, 12);
-  p1.moves.movePiece(5, 13);
-  p0.moves.movePiece(12, 11);
-  p0.moves.removePiece(20);
-  p1.moves.movePiece(13, 5);
-  p1.moves.removePiece(23);
-  p0.moves.movePiece(11, 12);
-  p1.moves.movePiece(15, 23);
-  p0.moves.movePiece(12, 11);
-  p0.moves.removePiece(23);
-  p1.moves.movePiece(22, 23);
-  p0.moves.movePiece(8, 15);
-  p1.moves.movePiece(23, 16);
-  p1.moves.removePiece(15);
-  p0.moves.movePiece(7, 15);
-  p1.moves.movePiece(16, 23);
+  p0.moves.placePiece(1);
+  p1.moves.placePiece(2);
+  p0.moves.placePiece(9);
+  p1.moves.placePiece(17);
+  p0.moves.placePiece(10);
+  p1.moves.placePiece(14);
+  p1.moves.removePiece(9);
+  p0.moves.placePiece(19);
+  p1.moves.placePiece(11);
+  p0.moves.placePiece(21);
+  p1.moves.placePiece(20);
+  p0.moves.movePiece(1, 9);
+  p1.moves.movePiece(14, 13);
+  p0.moves.movePiece(21, 22);
+  p1.moves.movePiece(13, 14);
+  p1.moves.removePiece(22);
+  p0.moves.movePiece(9, 1);
+  p1.moves.movePiece(20, 21);
   p0.moves.movePiece(1, 9);
-  p1.moves.movePiece(23, 16);
-  p1.moves.removePiece(15);
-  p0.moves.movePiece(11, 10);
-  p1.moves.movePiece(2, 1);
-  p0.moves.movePiece(10, 11);
-  p0.moves.removePiece(1);
-  p1.moves.movePiece(16, 23);
-  p0.moves.movePiece(9, 8);
-  p1.moves.movePiece(23, 16);
-  p1.moves.removePiece(8);
-  p0.moves.movePiece(11, 12);
-  p1.moves.movePiece(16, 23);
-  p0.moves.movePiece(12, 11);
-  p0.moves.removePiece(23);
-  p1.moves.movePiece(5, 13);
-  p0.moves.movePiece(0, 7);
-  p1.moves.movePiece(13, 5);
-  p1.moves.removePiece(7);
-  p0.moves.movePiece(11, 12);
-  p1.moves.movePiece(17, 16);
-  p0.moves.movePiece(12, 11);
-  p0.moves.removePiece(18);
-  p1.moves.movePiece(5, 13);
-  p0.moves.movePiece(3, 5);
-  p1.moves.movePiece(16, 23);
-  p0.moves.movePiece(5, 3);
-  p0.moves.removePiece(13);
-  p1.moves.movePiece(23, 13);
-  p0.moves.movePiece(11, 9);
-  p1.moves.movePiece(13, 5);
+  p1.moves.movePiece(14, 13);
+  p1.moves.removePiece(9);
```

Comment:
It seems like you are testing these corner cases of the rules here. Can you add comments for the moves that test that?

Reply:
The edge case happens just one time in this scenario